node/src/bin/manager.rs

```diff
@@ -344,6 +344,20 @@ pub enum Command {
         #[clap(long, short)]
         force: bool,
     },
+
+    // Deploy a subgraph
+    Deploy {
+        name: DeploymentSearch,
+        deployment: DeploymentSearch,
+
+        /// The url of the graph-node
+        #[clap(long, short, default_value = "http://localhost:8020")]
+        url: String,
```

Comment:
It would be awesome if we would _not_ need a `url`, but getting away from that might be pretty tough as we would have to instantiate our own `SubgraphRegistrar`; if the default is ok in most cases, I think it's not a big deal.

Reply:
That didn't cross my mind, just took a look at how that would turn out. We still would need to provide a NodeID when creating a new SubgraphRegistrar. 
And the default_value for the url i believe is almost always ok unless the indexer change it explicitly. So we can go with this now and update in the future?

Reply:
Yes, let's not hold this up because of the URL; just go ahead and merge